src/euphorie/client/browser/choice.py

```diff
@@ -0,0 +1,107 @@
+from euphorie.client import utils
+from euphorie.client.navigation import getTreeData
+from plone import api
+from plone.memoize.instance import memoize
+from Products.Five import BrowserView
+
+
+class IdentificationView(BrowserView):
+    """A view for displaying a choice in the identification phase."""
+
+    variation_class = "variation-risk-assessment"
+
+    @property
+    @memoize
+    def webhelpers(self):
+        return api.content.get_view("webhelpers", self.context.aq_parent, self.request)
+
+    def check_render_condition(self):
+        # Render the page only if the user can inspection rights,
+        # otherwise redirect to the start page of the session.
+        if not self.webhelpers.can_inspect_session:
+            return self.request.response.redirect(
+                "{session_url}/@@start".format(
+                    session_url=self.webhelpers.traversed_session.absolute_url()
+                )
+            )
```

Comment:
To make the code more readable I would add a `def redirect` method that would make this possible:
```suggestion
            return self.redirect("@@start")
```
or something similar, IDK....  `self.webhelpers.redirect_to_session_path` or whatever.
That can be useful also for the other view.

Reply:
Yes, but let's refactor later and also include `browser/risk.py` to keep things halfways consistent.